### PR TITLE
fix(flow): flow task service node complete time is not correct

### PR DESCRIPTION
**Pull request by krihy**
#### What type of PR is this?

type-bug
module-flow
#### What this PR does / why we need it:
1.fix flow completed time,  update flow status to `COMPLETED` after callback completed 
2.fix `ServiceTaskExecutingCompleteListener` status to `EXECUTING` , because service is not completed until callback task completed.
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/DBStructureComparisonFlowableTask.java [hunk 1]
```diff
@@ -285,6 +287,7 @@ protected void onSuccessful(Long taskId, TaskService taskService) {
         log.info("Structure comparison task succeed, taskId={}", taskId);
         try {
             taskService.succeed(taskId, taskResult);
+            super.onSuccessful(taskId, taskService);
             updateFlowInstanceStatus(FlowStatus.EXECUTION_SUCCEEDED);
             super.onSuccessful(taskId, taskService);
```
Comment (smallsheeeep) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/DBStructureComparisonFlowableTask.java:292]: Double invoke `super.onSuccessful(taskId, taskService)`?

Comment (krihy) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/DBStructureComparisonFlowableTask.java]: OKEY,  done

**Review by smallsheeeep — APPROVED** [review]
LGTM

**Review by guowl3 — APPROVED** [review]
LGTM

**Review by PeachThinking — APPROVED** [review]
LGTM

**Review by LuckyPickleZZ — APPROVED** [review]

File: server/odc-service/src/main/java/com/oceanbase/odc/service/flow/listener/ServiceTaskExecutingCompleteListener.java [hunk 2]
```diff
@@ -55,7 +55,7 @@ protected FlowNodeStatus doModifyStatusOnStart(FlowTaskInstance target) {
 
     @Override
     protected FlowNodeStatus doModifyStatusOnEnd(FlowTaskInstance target) {
-        return internalModify(target, FlowNodeStatus.COMPLETED);
+        return FlowNodeStatus.EXECUTING;
```
Comment (yhilmare) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/flow/listener/ServiceTaskExecutingCompleteListener.java:58]: you may return null instead, just a suggestion, the way you do is fine too

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**